src/relay.cpp

```diff
@@ -517,30 +665,37 @@ void relay_client(int sock, const uint8_t *msg, int32_t len, const ip6_hdr *ip_h
  * @return none
  */
 void relay_relay_forw(int sock, const uint8_t *msg, int32_t len, const ip6_hdr *ip_hdr, relay_config *config) {
-    static uint8_t buffer[BUFFER_SIZE];
-    dhcpv6_relay_msg new_message;
-    auto current_buffer_position = buffer;
     auto dhcp_relay_header = parse_dhcpv6_relay(msg);
 
-    if (dhcp_relay_header->hop_count >= HOP_LIMIT)
+    if (dhcp_relay_header->hop_count >= HOP_LIMIT) {
+        char addr_str[INET6_ADDRSTRLEN];
+        inet_ntop(AF_INET6, &ip_hdr->ip6_src, addr_str, INET6_ADDRSTRLEN);
+        syslog(LOG_INFO, "receive relay-forward message from %s with hop count %d over limit",
+               addr_str, dhcp_relay_header->hop_count);
```

Comment:
Should the wording be: "dropping relay-forward message"?

Reply:
sure, will update in latest diff.